### ISSUE TITLE
Installation of ST viewer

### ISSUE THREAD

**Issue by dalide** [jfnavarro/st_viewer#44]
Hi I tried to install STViewer on my mac, but it says it's not compatible with the version of the macOS, requiring maxOS 10.15 or later, as shown below

<img width="773" alt="Screen Shot 2020-04-29 at 11 46 43 AM" src="https://user-images.githubusercontent.com/16587569/80623070-4c737900-8a0f-11ea-9925-89ef716fcb3e.png">

so I tried to install the old version of ST Viewer, version 0.7.1 which was released in 2018, but it gave me the same error, which does not make sense, because macOS 10.15 is released in 2019. 

Could you help with this problem?  

thanks

**Comment by jfnavarro** [comment]
Hi, 

That message is strange indeed. The ST Viewer does not require
the version 10.15. There may be some strange configuration in your
system? 

Maybe you could try the the latest "old" release (0.7.6):

https://github.com/SpatialTranscriptomicsResearch/st_viewer/releases

**Comment by dalide** [comment]
> Hi,
> 
> That message is strange indeed. The ST Viewer does not require
> the version 10.15. There may be some strange configuration in your
> system?
> 
> Maybe you could try the the latest "old" release (0.7.6):
> 
> https://github.com/SpatialTranscriptomicsResearch/st_viewer/releases

Not sure what is going on on my macOS:( , there should be no any blocking.  I tried the version 0.7.6, it gave the same error message.

**Comment by dalide** [comment]
Actually I also tried a few other mac, they all the same, giving the same message.

**Comment by dalide** [comment]
Hi @jfnavarro I just found a way to open, right click -> open content package -> MacOS -> STViewer, it will open a terminal then the ST viewer window..

**Comment by dalide** [comment]
@jfnavarro Thanks for pointing out the version 0.7.6.  Actually I found out that the latest version 0.8.1 has a different interface for loading data and it requires spot coordinates, while in version 0.7.6, it is not required. And what happened was, I tried to open 0.8.1, and it says requiring macOS10.15, then after I close it, I did not notice there is a new Device called "ST Viewer" under the Devices in Finder,  so next time when I tried to open 0.7.6, or any older version, it by default used the ST viewer device, which is same as 0.8.1. So I need to manually turn off the device, and then reopen version 0.7.6, it is working now.  I am closing the issue.
